requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==1.1.3
+-rrequirements.txt
```

Comment:
I know this is how boto3 does it, but do you know why we include requirements.txt?   For docs we should only need the `pip install -e .` and the `requirements-docs.txt` file.  Do we want to be pulling in dev versions of botocore when building docs?